src/wayland/dmabuf/mod.rs

```diff
@@ -146,6 +449,7 @@ impl DmabufState {
         D: GlobalDispatch<zwp_linux_dmabuf_v1::ZwpLinuxDmabufV1, DmabufGlobalData>
             + BufferHandler
             + DmabufHandler
+            + DmabufFeedbackHandler
```

Comment:
You should probably note that with this constructor that the functions inside DmabufFeedbackHandler will not be invoked.

Reply:
done